backend/backend/Models/User.cs

```diff
@@ -23,6 +23,9 @@ public User()
 
         [Required]
         public string Password { get; set; }
+        
+        [Required]
```

Comment:
Please, kind sir, could you add salting to the constructor?

Reply:
I'd be glad to